src/twisted/newsfragments/9287.feature

```diff
@@ -0,0 +1,3 @@
+Add optional cwd argument to twisted.runner.procmon.ProcMon.addProcess
+
+Also deprecate direct introspection of ProcMon's processes: processes should not be directly accessed or pickled.
```

Comment:
this needs to be in a separate 9287.removal file, since this is  not about a feature.

see http://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles for info about how these fragments are used